1.5/Defs/ThingDefs_Misc/FroggeMask.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+    <ThingDef ParentName="HatMakeableBase">
+        <defName>Apparel_FroggeVisageMask</defName>
+        <label>frogge visage mask</label>
+        <description>A hard facial mask that resembles a stylized frogge face.</description>
+        <techLevel>Neolithic</techLevel>
+        <graphicData>
+            <texPath>Things/Pawn/Apparel/FroggeMask/FroggeMask</texPath>

```

Comment:
Prefix the textures with `MSS_FP_` as well.

Reply:
Renamed all textures and texpath to include prefix MSS_FP_